packages/app-desktop/gui/ResizableLayout/utils/useLayoutItemSizes.test.ts

```diff
@@ -20,6 +20,46 @@ describe('useLayoutItemSizes', () => {
 		expect(layout.isRoot).toBe(true);
 	});
 
+	test('should strecth the last visible child item if all siblings have fixed size and the last child is not visible', () => {
+		const layout: LayoutItem = validateLayout({
+			key: 'root',
+			width: 200,
+			height: 100,
+			direction: LayoutItemDirection.Row,
+			children: [
+				{ key: 'col1', width: 50 },
+				{ key: 'col2', width: 50 },
+				{ key: 'col3', width: 70, visible: false },
+			],
+		});
+
+		expect(layout.children).toEqual([
+			{ key: 'col1', width: 50 },
+			{ key: 'col2' },
+			{ key: 'col3', width: 70, visible: false },
+		]);
+	});
+
+	test('should strecth the last child item if all siblings have fixed size', () => {
+		const layout: LayoutItem = validateLayout({
+			key: 'root',
+			width: 200,
+			height: 100,
+			direction: LayoutItemDirection.Row,
+			children: [
+				{ key: 'col1', width: 50 },
+				{ key: 'col2', width: 50 },
+				{ key: 'col3', width: 70 },
+			],
+		});
+
+		expect(layout.children).toEqual([
+			{ key: 'col1', width: 50 },
+			{ key: 'col2', width: 50 },
+			{ key: 'col3' },
+		]);
+	});
```

Comment:
Laurent, is this what you meant in your second comment? I'm just not sure if I understood it right.